Vagrantfile

```diff
@@ -52,11 +52,12 @@ VERSION=$(lsb_release -sr | tr '[:upper:]' '[:lower:]')
 bash ${GIT_PATH}/.setup/install_worker.sh #{extra_command} 2>&1 | tee ${GIT_PATH}/.vagrant/install_worker.log
 SCRIPT
 
-box_extra = if Vagrant::Util::Platform.darwin? && (`uname -m`.chomp == "arm64" || (`sysctl -n machdep.cpu.brand_string`.chomp.include? 'M1'))
-  '-arm64'
-else
-  ''
-end
+base_boxes = Hash[]
+
+# Should all be base Ubuntu 20.04 boxes
```

Comment:
I would reword this to drop the "20.04" part of this, so that when we upgrade to 22.04 (and beyond), it's one less line that needs to change.

```suggestion
# Should all be base Ubuntu boxes that use the same version
```